x-pack/filebeat/docs/inputs/input-cel.asciidoc

```diff
@@ -1,876 +0,0 @@
-[role="xpack"]
-
-:type: cel
-:mito_version: v1.17.0
```

Comment:
This version is not reflected correctly in the markdown (for example [here](https://github.com/elastic/beats/pull/42897/files#diff-787354525c35f1b019d662b6e290dcbb2daa31b758d421990718a0055599a760R136)). This would suggest that there are other lines that are missing or incorrect.